tempto-core/src/main/java/com/teradata/tempto/fulfillment/table/hive/tpcds/ImmutableTpcdsTablesRequirements.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.teradata.tempto.fulfillment.table.hive.tpcds;
+
+import com.teradata.tempto.Requirement;
+import com.teradata.tempto.Requirements;
+import com.teradata.tempto.RequirementsProvider;
+import com.teradata.tempto.configuration.Configuration;
+
+import static com.teradata.tempto.fulfillment.table.TableRequirements.immutableTable;
+
+public class ImmutableTpcdsTablesRequirements
+        implements RequirementsProvider
+{
+    public static final Requirement CATALOG_SALES = immutableTable(TpcdsTableDefinitions.CATALOG_SALES);
```

Comment:
shouldn't this be coded similarly as `ImmutableTpchTablesRequirements` (e.g: requirement for each table)?